.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
It isn’t really meant to enforce anything so it’s fine to ignore it in suggested changes. I imagine a next person would fix any outliers or we'd configure prettier (unsure of you can for markdown?). Anyways, I don’t feel strongly about this, so I’m fine with removing it if you so wish 👍

Reply:
Oh wait, I think I better understand what you’re thinking of re suggested changes, you want it to be a single line so you can make atomic suggestions, right?

Reply:
> Anyways, I don’t feel strongly about this, so I’m fine with removing it if you so wish 👍

I think it would be better to remove it as probably the better way to enforce a 'code format' would be via `yarn format` (which I think is already in place). Also, introducing this I think will force the changelog.md to follow the same format pattern which may result into a massive PR to just adapt to it.

> Oh wait, I think I better understand what you’re thinking of re suggested changes, you want it to be a single line so you can make atomic suggestions, right?

Yeah, if phrases are single lines it's easier to do GH inline suggestions via the PR review interface :3

Reply:
> Also, introducing this I think will force the changelog.md to follow the same format pattern which may result into a massive PR to just adapt to it.

No that was what I was trying to convey, this is not an enforcement, just a visual guide.

Anyhoo, will remove, as well as newlines! 👍

Reply:
> this is not an enforcement, just a visual guide.

oh so this would not "format" the .md files? (like as it if was a prettier config?)
Then how did you break the lines in the .md? manually? 🤔

Reply:
Yup :)

I just removed the newlines btw. The one thing that comes to mind is that bullet lists are now slightly less readable in plain text format, due to missing indentation.